XML/DTD.sublime-syntax

```diff
@@ -0,0 +1,12 @@
+%YAML 1.2
+---
+name: DTD
+file_extensions:
+  - dtd
+  - ent
+  - mod
+scope: text.xml.dtd
+
+contexts:
+  main:
+    - include: scope:text.xml#dtd
```

Comment:
Let's use an explicit syntax path here since it will only work with the XML syntax we ship.